examples/pulse_control/6_Use_jobs_with_pulse_sequence_sweeps.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>Our code will reuse large parts of the first <a href="https://github.com/aws/amazon-braket-examples/raw/161b36cc401132f5baba3dc9f43db16503bfdbca/examples/pulse_control/1_Bringup_experiments.ipynb" target="_blank">notebook</a>.  -&gt; Instead of first, use the notebook name here.</p><p> </p><p>The main code of the algorithm is located in a <a href="https://github.com/aws/amazon-braket-examples/raw/161b36cc401132f5baba3dc9f43db16503bfdbca/examples/pulse_control/qubit_spectroscopy/qubit_spectroscopy_sweep.py" target="_blank">python file</a> and consists of generating a single-pulse <code>PulseSequence</code> with a varying frequency. The script is parametrized by the following parameters:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>